app/views/mentoring/automation/tooltip_locked.html.haml

```diff
@@ -1,21 +1,20 @@
-.heading.c-generic-tooltip.text-14.w-413
-  %h4.font-semibold.mx-24.mt-20.mb-4 What is automation?
-
-  -# %p.mx-24
-    = ReactComponent::
-    Requires a satisfaction percentage of
-    %strong= @min_satisfaction_percentage.to_s
-    (current: #{@satisfaction_percentage})
-
-  -# %p.mx-24
+.p.heading.c-generic-tooltip.text-14.some{style:'padding:16px 24px;width:413px;'}
```

Comment:
```suggestion
.p.heading.c-generic-tooltip.text-14.some{style:'padding:16px 24px;width:413px;'}
```

1. What's `.p`? 
2. Also use `py-16 px-24` rather than add styling.
3. use `class: "w-[413px]"` rather than `style`

Reply:
Oops, `p` is a chunk I left behind, it was padding, that didn't work because the padding set in c-generic-tooltip class is more "!important". that is why I used inline styling. I tried it with class, with `!py-16`, `!py-[16px]`, but the only way I could force it to work was using inline styling.

About `.some` it is super embarrassing to admit, but I read the answer first here: 
https://stackoverflow.com/questions/8090010/haml-how-to-set-inline-style-for-element-in-haml
and immediately thought it was a pseudoselector-something in haml to insert inline styling..
but now that I read the question too, I see that this is only a random class.. sorry about this.

Reply:
Cool cool!